subworkflows/nf-core/fasta_binning_concoct/main.nf

```diff
@@ -0,0 +1,56 @@
+include { CONCOCT_CUTUPFASTA           } from '../../../modules/nf-core/concoct/cutupfasta/main.nf'
+include { CONCOCT_CONCOCTCOVERAGETABLE } from '../../../modules/nf-core/concoct/concoctcoveragetable/main.nf'
+include { CONCOCT_CONCOCT              } from '../../../modules/nf-core/concoct/concoct/main.nf'
+include { CONCOCT_MERGECUTUPCLUSTERING } from '../../../modules/nf-core/concoct/mergecutupclustering/main.nf'
+include { CONCOCT_EXTRACTFASTABINS     } from '../../../modules/nf-core/concoct/extractfastabins/main.nf'
+
+workflow FASTA_BINNING_CONCOCT {
+
+    take:
+    ch_fasta // channel: [ val(meta), [ fasta ] ]       // raw contigs from assembly
+    ch_bam   // channel: [ val(meta), [ bam ], [bai]]   // bam files of original FASTQ Files mapped back to each contig. meta must correspond to ch_fasta
+
+    main:
+    ch_versions = Channel.empty()
+
+    // required to create bedfile due to coverage table
+    produce_bedfile = true
```

Comment:
Should this not be a value declared in the `input`?

Reply:
It's _required_ for the subworkflow i.e., will error without it), so I decided to set it here